### PR TITLE
#21943 Fix ECR Vulnerabilities: thecodingmachine/gotenberg

### PR DESCRIPTION
**Pull request by SaPH17**
### Changes
- Update gotenberg version to 8.3.0
- Vulcan PR https://github.com/vulcan-ai/vulcan/pull/22032

### CODE REVIEW

File: charts/datasaur/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.11
+version: 2.1.12
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
```
Comment (tonoman3g) [inline charts/datasaur/Chart.yaml:21]: bagian ini `appVersion: ">=6.56.0"`, nanti diganti jadi sekitar `>=6.59.0` ya

@SaPH17

Comment (SaPH17) [reply charts/datasaur/Chart.yaml:21]: done ya ko

Comment (SaPH17) [reply charts/datasaur/Chart.yaml:21]: eh, kayaknya zip yang 2.1.12 kesenggol ya?
ku revert bentar ya

**Comment by daniel-jeremia** [comment]
@SaPH17 barusan ada merge ke master, boleh tolong resolve conflict ya, bump up aja version nya. Thanks!

**Comment by SaPH17** [comment]
done ya @daniel-jeremia

**Comment by michael-jong-ds** [comment]
jgn lupa di reindex ya
`helm repo index --url https://datasaur-ai.github.io/helm-charts .`

**Review by michael-jong-ds — APPROVED** [review]
LGTM

**Review by daniel-jeremia — APPROVED** [review]
LGTM

**Pull request merged**